sky/dashboard/src/components/volumes.jsx

```diff
@@ -350,24 +365,35 @@ function VolumesTable({
                   <TableCell className="font-medium">{volume.name}</TableCell>
                   <TableCell>{volume.infra || 'N/A'}</TableCell>
                   <TableCell>
-                    <span
-                      className={`inline-flex items-center px-2.5 py-0.5 rounded-full text-sm font-medium ${
-                        volume.status === 'READY'
-                          ? 'bg-green-100 text-green-800'
-                          : volume.status === 'IN_USE'
-                            ? 'bg-blue-100 text-blue-800'
-                            : 'bg-gray-100 text-gray-800'
-                      }`}
-                    >
-                      {volume.status || 'UNKNOWN'}
-                    </span>
+                    <StatusBadge status={volume.status} />
                   </TableCell>
                   <TableCell>{formatSize(volume.size)}</TableCell>
                   <TableCell>{volume.user_name || 'N/A'}</TableCell>
                   <TableCell>
                     {formatTimestamp(volume.last_attached_at)}
                   </TableCell>
                   <TableCell>{volume.type || 'N/A'}</TableCell>
+                  <TableCell>
+                    {Array.isArray(volume.usedby_clusters) &&
+                    volume.usedby_clusters.length > 0
+                      ? volume.usedby_clusters.map((cluster, idx) => (
+                          <span key={cluster}>
+                            <Link
+                              href={`/clusters/${encodeURIComponent(cluster)}`}
+                              className="text-sky-blue hover:underline"
+                            >
+                              {cluster}
+                            </Link>
+                            {idx < volume.usedby_clusters.length - 1
+                              ? ', '
+                              : ''}
+                          </span>
+                        ))
+                      : Array.isArray(volume.usedby_pods) &&
+                          volume.usedby_pods.length > 0
+                        ? volume.usedby_pods.join(', ')
+                        : 'N/A'}
```

Comment:
How does this look when there's few 10s of pods? If more than 1 pod/cluster is using the volume?